src/solvers/dgsem/calc_volume_integral.jl

```diff
@@ -79,6 +86,41 @@ function calc_volume_integral!(du, u, mesh,
     return nothing
 end
 
+function calc_volume_integral!(du, u, mesh,
+                               nonconservative_terms, equations,
+                               volume_integral::VolumeIntegralAdaptive{VolumeIntegralWeakForm,
+                                                                       VolumeIntegralFD,
+                                                                       Indicator},
+                               dg::DGSEM,
+                               cache) where {
+                                             VolumeIntegralFD <:
+                                             VolumeIntegralFluxDifferencing,
+                                             Indicator <: AbstractIndicator}
+    @unpack volume_integral_default, volume_integral_stabilized, indicator = volume_integral
+
+    # Calculate decision variable
+    decision = @trixi_timeit timer() "integral selector" indicator(u, mesh, equations,
+                                                                   dg, cache)
```

Comment:
```suggestion
    decision = @trixi_timeit timer() "blending factors" indicator(u, mesh, equations,
                                                                  dg, cache)
```

For consistency with the Hennemann-Gassner shock capturing?